docs/dle602-a3/REPRODUCIBILITY_REPORT.md

```
@@ -0,0 +1,151 @@
+### Lightweight Archive
+
+**Setup & Installation**:
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt -c constraints-a3.txt
+```
+✅ Clean installation; all dependencies resolve without conflict.
+
+**Test Results**:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)